src/test/java/lotto/domain/WinningTicketTest.java

```diff
@@ -4,39 +4,41 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-import java.util.Arrays;
-import java.util.List;
-import java.util.stream.Collectors;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.assertj.core.api.Assertions.assertThat;
 
-public class WinningTicketTest {
+class WinningTicketTest {
 
     @Test
-    void 로또번호와_당첨결과_비교() {
-        int[] value = {1, 2, 3, 4, 5, 6};
-        List<LottoBall> winningBallValues = Arrays.stream(value).mapToObj(LottoBall::new).collect(Collectors.toList());
-        WinningTicket winningTicket = new WinningTicket(winningBallValues, 7);
+    @DisplayName("몇개 맞추는지 test")
+    void hitLottoBallTest() {
+        LottoTicket lottoTicket = new LottoTicket("1,2,3,4,5,6");
```

Comment:
전체적으로 `new LottoTicket` 대신 `LottoTicket.of` 로 해야 되는것 같은데요? :) 
```suggestion
        LottoTicket lottoTicket = LottoTicket.of("1,2,3,4,5,6");
```